packages/url_launcher/url_launcher/lib/src/link.dart

```diff
@@ -0,0 +1,135 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/widgets.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:url_launcher_platform_interface/link.dart';
+import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
+
+export 'package:url_launcher_platform_interface/link.dart'
+    show LinkTarget, LinkWidgetBuilder, FollowLink;
```

Comment:
I don't think you need to export from this package; you only need to import these 3 classes to use them here, right?

Reply:
These 3 classes are needed by users. Here are examples of where they may need them:
```dart
Link(
  uri: Uri.parse('http://example.com/foo'),
  target: /**/LinkTarget/**/.blank,
  builder: (BuildContext context, /**/FollowLink/**/ followLink) {
    return Container();
  },
);
```
And in case they need to have a wrapper around the Link widget that takes the `builder` as an argument:
```dart
class GoogleLink extends StatelessWidget {
  GoogleLink({this.builder});

  final /**/LinkWidgetBuilder/**/ builder;

  final Uri uri = Uri.parse('https://google.com');

  @override
  Widget build(BuildContext context) {
    return Link(uri: uri, builder: builder);
  }
}
```

Reply:
Or maybe you mean that I can put this export in `lib/url_launcher.dart` directly instead of exporting them from here? I feel stupid now lol.